.github/ISSUE_TEMPLATE/ticket-story.md

```diff
@@ -5,11 +5,11 @@ about: A basic template for writing tickets in the form of user stories to hand
 title: ''
 labels: ''
 assignees: ''
-
 ---
 
 ### Description
-<!-- 
+
+<!--
```

Comment:
It is fine to include these minor changes formatting here, but bc it doesn't really relate to the scope of this ticket, can we exclude it in the future to minimize amount of code reviews are looking at? if want to update formatting of things, feel free to make a new ticket